tests/kafkatest/services/console_consumer.py

```diff
@@ -102,6 +100,10 @@ def __init__(self, context, num_nodes, kafka, topic, group_id="test-consumer-gro
         for node in self.nodes:
             node.version = version
 
+        self.jmx_tool = JmxTool(context, jmx_object_names=jmx_object_names, jmx_attributes=(jmx_attributes or []),
+                                root=ConsoleConsumer.PERSISTENT_ROOT) \
+                        if jmx_object_names is not None else None
```

Comment:
Should we check `jmx_attributes` as well?

Reply:
I believe we only want to check `jmx_object_names` since it seems like it's the only required parameter for this JmxTool wrapper.

Reply:
correction: both are required here. we should be checking both

Reply:
This syntax is a bit hard to follow with the conditional at the end. Can you rewrite it to something like:

```python
self.jmx_tool = None
if jmx_object_names is not None:
    self.jmx_tool = ...
```